databend/create.sql

```diff
@@ -106,4 +106,4 @@ CREATE TRANSIENT TABLE hits
     URLHash BIGINT NOT NULL,
     CLID INTEGER NOT NULL
 )
-CLUSTER BY (CounterID, EventDate, UserID, EventTime, WatchID);
+CLUSTER BY (CounterID, EventDate, UserID, EventTime, WatchID) COMPRESSION = "lz4" STORAGE_FORMAT = "native";
```

Comment:
is it actually correct to consider this configuration default? tuned configurations should be separated from "vanilla" set ups

Reply:
Because this test is under `storage.type = fs`, but databend is mainly optimized for cloud object storage (default zstd compression, parquet format storage), also the `config.toml` basically does not have too many additional settings.

Compared to other db `DISTRIBUTED BY HASH(UserID ) BUCKETS 192`, these configurations are more like specifying partition keys and engines.

It's easy to use adaptive default configurations by `storage.type`, maybe in the next version a few weeks later.

I don't want this to be a tuning result, so I set this pr to draft.
If you don't think this is a tuning result now, you can merge it, or wait for our next update. Thank you.